Tests/PerformanceTestPlan.xctestplan

```diff
@@ -54,6 +54,8 @@
         "NoImageTests",
         "OnboardingTests",
         "OpeningScreenTests",
+        "PerformanceTests\/testPerfTabs_3_20tabTray()",
```

Comment:
@jjSDET Please add a jira / github issue so that we can track the problem happening with these tests since there may be a legit bug to fix. 
Thanks!